src/lib/util/Config.ts

```diff
@@ -88,6 +88,18 @@ const DEFAULT_CONFIG: ConfigContent = {
 			],
 			disabled: false
 		},
+		{
+			name: ENGINE.ESLINT_CUSTOM,
+			targetPatterns: [
+				// Reduce default to be only what the other eslint engines can do (i.e. js and ts files).
+				// See https://github.com/forcedotcom/sfdx-scanner/issues/1146
+				// If user wants other files, then they update this inside of their ~/.sfdx-scanner/Config.json file
```

Comment:
@stephen-carter-at-sf , is there a plan to document this?

Reply:
Basically before we allowed all files to be picked up by eslint when --estlintconfig … was supplied. This seemed to be against customer expectations. So not restricting it to only ts and js files (excluding node_modules) by default. And then just like with other engines (since we don’t document them there either) users could theoretically change this list of target patterns from their ~/.sfdx-scanner/Config.json file. Because we don't document this configurability on the other engines today, I don't think we should do so for the custom eslint engine case either. We would either document them all (revealing the Config.json schema) or not document them all. For our future version of code analyzer (i.e. 5.x) we definitely should revisit this.